modules/cli/src/main/scala/scala/cli/commands/Package.scala

```diff
@@ -274,6 +294,9 @@ object Package extends ScalaCommand[PackageOptions] {
         else
           s"Wrote $dest"
       }
+
+    val mTimeDestPathOpt = if (!packageOptions.isDockerEnabled) Some(os.mtime(destPath)) else None
```

Comment:
Maybe we could check `packageType` instead of `packageOptions.isDockerEnabled` (as the former is actually used above to decide to build a docker image or not), and we could also change `PackageType.runnable` to an `Option[Boolean]`, that would be empty if the packaging type doesn't build a file (so `PackageType.Docker.runnable` would be `None`), and use that here as a check: `if (packageType.runnable.isEmpty) None else Some(…)`